addons/binding/org.openhab.binding.knx/ESH-INF/thing/basic.xml

```diff
@@ -0,0 +1,372 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="knx"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Generic KNX Actor -->
+	<thing-type id="basic">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="ip" />
+			<bridge-type-ref id="serial" />
+		</supported-bridge-type-refs>
+
+		<label>Basic KNX actor</label>
+		<description>An addressable basic KNX actor</description>
+
+		<config-description>
+			<parameter name="address" type="text">
+				<label>Address</label>
```

Comment:
Layout looks broken (most likely a combination of tabs and spaces), please format using eclipse